### ISSUE TITLE
Can we remove the cache directory before pack the image?

### ISSUE THREAD

**Issue by swapwz** [Azure/sonic-buildimage#10101]
#### Description

I found there is a directory named .cache under the path /root in my device. It will size about ~45MB disk. 
Can we remove the cache directory before pack the image?

#### Steps to reproduce the issue:
1. make target/sonic-broadcom.bin
2. you will found the .cache under the fsroot/root

#### Describe the results you received:
root@build:/home/xxx/build/202012/sonic-buildimages/fsroot/root/.cache# du -sh .
45M

#### Describe the results you expected:
there is no such directory under the fsroot/root.

#### Output of `show version`:

```
N/A
```
#### Output of `show techsupport`:

```
N/A
```

**Comment by zhangyanzhao** [comment]
@xumia please help to take a look. Thanks.

**Comment by xumia** [comment]
@swapwz , it only takes some space, right?
I am not expert, only some search. Found an answer for it, "generally safe, but not recommend", see https://askubuntu.com/questions/102046/is-it-okay-to-delete-the-cache-folder

Do you have any better answers, to make sure which one can be removed, or all can be removed?

**Comment by swapwz** [comment]
@xumia yes, it causes the image file become larger and takes some space after installed.
I think the fsroot/root/.cache/pip is useless after build the rootfs.  see
### https://stackoverflow.com/questions/9510474/removing-pips-cache

should we add pip clean up step,  as following:
```
## Clean up pip cache
sudo LANG=C chroot $FILESYSTEM_ROOT pip cache purge
sudo LANG=C chroot $FILESYSTEM_ROOT pip3 cache purge
```

**Comment by xumia** [comment]
Fix on master branch: https://github.com/Azure/sonic-buildimage/pull/10143

**Comment by swapwz** [comment]
> Fix on master branch: #10143 @swapwz , could you please help review it?

@xumia sure, how can I join it?

**Comment by xumia** [comment]
@swapwz ,you can open the link below, and add your comments, or you can approve it.
```
https://github.com/Azure/sonic-buildimage/pull/10143
```
